src/components/FormSections/PersonalInfo.jsx

```diff
@@ -320,12 +320,12 @@ function PersonalInfo(props){
                         <Label className="text-bold" htmlFor="phone-number">
                             {phoneNumberField.label}{(telephoneFieldState.required === "1") && <span className='required-text'>*</span>}
                         </Label>
-                        <span className="usa-hint" id="date-of-birth-hint">{phoneNumberField.help_text}</span>
+                        <span className="usa-hint" id="phone-number-hint">{phoneNumberField.help_text}</span>
                         <TextInput
                             data-test="phoneNumber"
                             id="phone-number"
                             className="radius-md"
-                            aria-describedby="phone-number_error"
+                            aria-describedby="phone-number-hint"
```

Comment:
@WilliamYKwon how does this change affect the announcement of the error message if it is displayed?

Reply:
Oh, it announces the hint instead even after the error msg is displayed. Sorry I didn't think of that, I'll make the changes!